R/data-importer-configuration.R

```diff
@@ -35,11 +35,11 @@ DataImporterConfiguration <- R6::R6Class(
       private$.setColumnUnit(column = column, value = value)
     },
 
-    #' @field timeUnitFromColumn If `TRUE`, units of the values in time column
+    #' @field isTimeUnitFromColumn If `TRUE`, units of the values in time column
     #' are defined in the column `timeUnit`. If `FALSE`, the unit is defined by
-    #' `timeUnit`.
-    timeUnitFromColumn = function(value) {
-      column <- private$.timeColumn
+    #' the value of `timeUnit`.
+    isTimeUnitFromColumn = function(value) {
```

Comment:
@msevestre You probably missed it (or it wasn't pushed, or got eaten by GitHub, whatever). I am naming this field now with `isXXX` to make clear it returns a logical.

Reply:
ok great